README.md

```diff
@@ -73,7 +73,7 @@ error_analyzer = ErrorAnalyzer(model, feature_names=feature_names)
 error_analyzer.fit(X_test, y_test)
 
 # print metrics regarding the Model Performance Predictor
-print(error_analyzer.mpp_summary(X_test, y_test, output_dict=False))
+print(error_analyzer.evaluate(X_test, y_test, output_format='text'))
```

Comment:
You should change `evaluate` to return the string (for now it prints it).
Also nitpick, we do not need not have `output_format='text'` as it is the default (same on l.136)